### DISCUSSION TITLE
Using Yazi as the file picker for Firefox

### DISCUSSION THREAD

**Discussion by nik-rev** [sxyazi/yazi#1078] [1. Q&A] [answered]
Hello, I'm using Firefox's file manager to select files on my computer or folders. I would like to use Yazi for this task. Is this something that can be done?

**Comment by sxyazi** [comment, marked as answer]
Sounds like https://github.com/GermainZ/xdg-desktop-portal-termfilechooser is what are you looking for.

**Reply by nik-rev** [reply]
Thanks
